scannerlib/configchecks/file_check.go

```diff
@@ -39,6 +39,9 @@ import (
 // MaxNonCompliantFiles is the maximum number of non-compliant files to be displayed for a single finding.
 const MaxNonCompliantFiles = 10
 
+const PipelineToken = "%%pipeline%%"
```

Comment:
Add a comment to explain what this is.

Also add a comment to the scan_instructions.proto that the path can be replaced with the last SQL query's output for this wildcard value, e.g. on top of here:
https://github.com/google/localtoast/blob/master/scannerlib/proto/scan_instructions.proto#L201

Reply:
Done.